shared/AppInsightsCore/src/JavaScriptSDK.Tests/Selenium/ApplicationInsightsCore.Tests.ts

```diff
@@ -190,11 +190,11 @@ export class ApplicationInsightsCoreTests extends TestClass {
                 const channelPlugin = new ChannelPlugin();
                 const appInsightsCore = new AppInsightsCore();
                 appInsightsCore.initialize({ instrumentationKey: expectedIKey }, [channelPlugin]);
-                const validateStub = this.sandbox.stub(appInsightsCore, "_validateTelmetryItem");
+                const validateStub = this.sandbox.stub(appInsightsCore.baseCore, "_validateTelmetryItem");
 
                 // Act
                 const bareItem: ITelemetryItem = { name: 'test item' };
-                appInsightsCore.track(bareItem);
+                appInsightsCore.baseCore.track(bareItem);
```

Comment:
Call appInsightsCore.track here, this is main scenario and we want to be sure it works fine